diff --git a/node_modules/pdf-parse/lib/pdf-parse.js b/node_modules/pdf-parse/lib/pdf-parse.js
index 0375ea9..d394e50 100644
--- a/node_modules/pdf-parse/lib/pdf-parse.js
+++ b/node_modules/pdf-parse/lib/pdf-parse.js
@@ -10,7 +10,6 @@
 
 //see https://github.com/mozilla/pdf.js/blob/master/src/display/api.js
 async function PDF(dataBuffer, options) {
-	let isDebug = (options && options.debug) ? options.debug : false;
 	let version = 'default';
 	if (options && options.version) {
 		version = options.version;
@@ -43,10 +42,6 @@
 			}
 			//get all pages text
 			let lastPromise; // will be used to chain promises
-			if(isDebug) {
-				let output = await PDFJS.getDocument( './test/data/05-versions-space.pdf' );
-				console.log(output);
-			}
 			lastPromise = page.getTextContent(options)
 				.then(function (content) {
 					//covert content to simple text

